frontend/src/pages/groups/GroupForm.tsx

```diff
@@ -63,8 +69,9 @@ const GroupForm: FunctionComponent<Props> = (props) => {
         <TextField
           label="Town or city"
           name="primaryLocation.townCity"
-          register={register}
           required
+          register={register}
+          errors={errors}
```

Comment:
I think there's an opportunity to create a wrapper component that passes those props automatically, but this PR is already monstrous so I held off.